ckanext/data_qld_theme/fanstatic/resource_file_upload.js

```diff
@@ -1,26 +1,42 @@
 jQuery(document).ready(function () {
```

Comment:
Add some general comments to help explain the business logic and reasons
Remove extra blank lines etc
Run lint and format the code